package.json

```diff
@@ -26,12 +30,11 @@
     "optimize:svg": "find assets -type f -name \"*.svg\" -print0 | xargs -0 -n 1 -P 6 svgo --multipass --pretty --enable=removeTitle",
     "optimize:png": "find assets -type f -name \"*.png\" -print0 | xargs -0 -n 1 -P 6 optipng",
     "watch:css": "chokidar \"components/**/*.scss\" \"pages/**/*.scss\" \"scss/**/*.scss\" -c \"npm run build-client:scss\"",
-    "watch:js": "chokidar \"app.jsx\" \"main.jsx\" \"js/**/*.js\" \"components/**/*.js?(x)\" \"pages/**/*.js?(x)\" \"webpack.config.js\" -c \"npm run build-client:js\"",
+    "watch:js": "chokidar \"app.jsx\" \"main.jsx\" \"js/**/*.js\" \"components/**/*.js?(x)\" \"pages/**/*.js?(x)\" \"webpack.**.js\" -c \"npm run build-js:dev:client\"",
     "test": "run-s test:**",
-    "test:eslint": "eslint --config ./.eslintrc.json  \"*.js\" \"*.jsx\" \"js/**/*.js\" \"components/**/*.jsx\" \"pages/**/*.jsx\" \"pages/**/*.js\" webpack.config.js",
+    "test:eslint": "eslint --config ./.eslintrc.json  \"*.js\" \"*.jsx\" \"js/**/*.js\" \"components/**/*.jsx\" \"pages/**/*.jsx\" \"pages/**/*.js\" webpack.**.js",
     "test:scss:styleguide:color": "stylelint \"components/**/*.scss\" \"pages/**/*.scss\" \"scss/**/*.scss\" \"!scss/foundation-styles/_variables.scss\" \"!scss/**/variables.scss\" \"!scss/overrides/_variables.scss\" --syntax scss --config .stylelintrc-colors.js",
-    "test:scss": "stylelint --config .stylelintrc \"components/**/*.scss\" \"pages/**/*.scss\" \"scss/**/*.scss\" --syntax scss",
-    "postinstall": "npm run build"
```

Comment:
replaced by `heroku-postbuild` task

Reply:
Let's roll this particular change back, given that staging doesn't work right now. And then we can always optimize this later.

Reply:
kk... reverted... sort of.

I have to make `postinstall` build with `production` version of webpack config for Heroku apps to run in the right mode (e.g., `production`). This change does introduce a side effect -- if you run `npm install` locally, Pulse will get built in `production` mode which is not what we want. I'll file a follow-up ticket so this PR doesn't grow any bigger.

Reply:
good call